tensorflow/python/framework/random_seed.py

```diff
@@ -25,6 +25,9 @@
 
 _DEFAULT_GRAPH_SEED = 87654321
 
+def _truncate_seed(seed):
+  return seed % 2147483647 # truncate to fit into 32-bit integer
```

Comment:
Replaced it with a constant. Couldn't find a pre-defined constant for this in Python....